snort is missing deps

build aborts with 

Package snort-pgsql is missing dependencies for the following libraries:
liblzma.so.5
make[3]: *** [/home/buildbot/slave-local/malta_le/build/bin/malta/packages/packages/snort-pgsql_2.9.7.2-1_malta_mipsel.ipk] Error 1

this happens for all 3 variants
   ERROR: package/feeds/packages/snort failed to build (build variant: basic).
   ERROR: package/feeds/packages/snort failed to build (build variant: mysql).
   ERROR: package/feeds/packages/snort failed to build (build variant: pgsql).

--- a/net/snort/Makefile
+++ b/net/snort/Makefile
@@ -83,7 +83,8 @@ CONFIGURE_ARGS += \
 	--with-libpcre-libraries="$(STAGING_DIR)/usr/lib" \
 	--with-daq-includes="$(STAGING_DIR)/usr/include" \
 	--with-daq-libraries="$(STAGING_DIR)/usr/lib" \
-	--disable-static-daq
+	--disable-static-daq \
+	--disable-lzma
 
 CONFIGURE_VARS += \
 	CPPFLAGS="$$$$CPPFLAGS -I$(STAGING_DIR)/usr/include/mysql" \

